### ISSUE TITLE
[BUG] Notification for AttributeChangeRequest cannot be opened directly

### ISSUE THREAD

**Issue by kulla** [nmshd/feedback#5]
**Describe the bug**
After sending a `AttributesChangeRequest` a notification occurs in the Enmeshed app. Clicking on it does not open the send message (like I am used to for notifications of messaging apps).  

**To Reproduce**

1. Create a relationship to an Enmeshed app. 
2. Send a `AttributesChangeRequest` to the client.
3. Notification in the Enmeshed app cannot be opened directly.

For the `AttributesChangeRequest` we used something like:

```ts
 const client = ConnectorClient.create({ ... })

await client.messages.sendMessage({
      recipients: [session.enmeshedId],
      content: {
        '@type': 'RequestMail',
        to: [session.enmeshedId],
        cc: [],
        subject: '<Subject>',
        body: '<Body>',
        requests: [
          {
            '@type': 'AttributesChangeRequest',
            attributes: [{ name: "foo", value: "bar" }],
            applyTo: session.enmeshedId,
            reason: '<reason>',
          },
        ],
      },
})
```

**Expected behavior**
Clicking on the notification will open the message in the Enmeshed App.

**Screenshots**
Notification in the app after sending the `AttributesChangeRequest`:

![Screenshot_20220103-181648](https://user-images.githubusercontent.com/1327215/147960085-b9a386de-ff1d-44d4-a5e1-94305713a122.png) 

**Environment**
 - Product: App
 - Version: 1.6.6
 - Device: Android 10

**Comment by jkoenig134** [comment]
I can reproduce this, it's on our board.

**Comment by stnmtz** [comment]
Hi @kulla,

thanks again for your report. This should be fixed with App v1.8.0, expected in stores next week. I'll leave the issue open until you can confirm.

Best Regards
